ultralytics/engine/trainer.py

```diff
@@ -239,16 +239,16 @@ def calc_weights(self):
             print("Not using class weights")
         elif isinstance(cls_weights, list) and len(cls_weights) == nc:
             cls_weights = torch.Tensor(cls_weights)
-        elif cls_weights == "median":
-            cls_weights = self.get_median_frequency_weights(nc)
+        elif cls_weights == "median-inverse":
```

Comment:
this feels wordy

Reply:
fair enough, I just want to indicate that the two methods are really identical except for the numerator , i.e. median-count vs total-count / nc

Reply:
check out the new options